### ISSUE TITLE
unhealthy loss

### ISSUE THREAD

**Issue by Jervint** [noahchalifour/rnnt-speech-recognition#14]
@noahchalifour I have trained the model with the hparams you get. But I got the loss seemed abnormal. Can you give me some advice, thanks!
![image](https://user-images.githubusercontent.com/20737207/77977337-08b72200-7332-11ea-81d7-0435fde7715d.png)

**Comment by noahchalifour** [comment]
Please try again with the latest commit (01edf390be6b7cc52b7522d35a985c9b18419fa2 as of right now), I have made a lot of changes to the model and training. Feel free to re-open if issue persists.

**Comment by digital10111** [comment]
I am guessing we are referring to the non convergence of the loss here as unhealthy?
What change did you make @noahchalifour, for this? Did you find out something which was causing this?

**Comment by Jervint** [comment]
I trained mode with your new commit, the issue still exists. After several epochs, the loss will grow to NaN, and the WER/CER/Accuray don't have little improvements in training. Could you share an available pre-trained model even though its accuracy is unfavorable.  Thanks! @noahchalifour 
![image](https://user-images.githubusercontent.com/20737207/78520122-179d4900-77f8-11ea-8428-820f9e6790b2.png)

**Comment by thebeyonder001** [comment]
Did you guys solve it?
